src/__deprecated__/components/form/form-button/form-button.spec.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import 'jest-styled-components';
-import { rootTagTest } from '../../../utils/helpers/tags/tags-specs/tags-specs';
+import { rootTagTest } from '../../../../utils/helpers/tags/tags-specs/tags-specs';
 import FormButton from './form-button.component';
-import Button from '../../button';
+import Button from '../../../../components/button/button.component';
```

Comment:
I don't think `/button.component` is needed here. The `src/components/button/` directory has an `index.js` file which `export`s the `Button` from `./button.component`.

```suggestion
import Button from '../../../../components/button';
```